.stylelintrc.json

```diff
@@ -1,6 +1,19 @@
 {
   "extends": ["stylelint-config-standard"],
   "rules": {
-    "no-descending-specificity": null
+    "alpha-value-notation": null,
```

Comment:
is there a reason so many rules need to be disabled? generally it's bad practice to change lint rules to fit things in

also the indentation here looks strange

Reply:
Completely understand, however, the default ruleset is very opinionated to force a specific style. Something that would be easier for multiple maintainers would be to use a combination of eslint, stylelint and prettier. Since linters usually contain not just code quality rules but also stylistic rules. Prettier can be used for formatting concerns and linters for code quality.